### ISSUE TITLE
Rails/HttpPositionalArguments raising on draw route

### ISSUE THREAD

**Issue by wbotelhos** [rubocop-hq/rubocop#5624]
The Cop `Rails/HttpPositionalArguments` identifies a false positive situation:

## Expected behavior

```
routes.draw do
  post :create, to: 'anonymous#create'
end
```

Should be ok, since the wrapper is a `draw` method.

## Actual behavior

It warns to use params over positional arguments. But `params` is used only to call, not to declare.

## RuboCop version

```
0.52.1 (using Parser 2.5.0.2, running on ruby 2.5.0 x86_64-darwin17)
```

**Comment by jesperronn** [comment]
Was this ever solved? I just ran into the same issue on Rubocop 0.80.

I am OK with the cop, but I would REALLY like to have a workaround for `routes.draw`

**Comment by jesperronn** [comment]
BTW, this seems to be a duplicate of https://github.com/rubocop-hq/rubocop/issues/3629

**Comment by korny** [comment]
No, it still happens for us.
